src/views/certifications/types.ts

```diff
@@ -1,5 +1,14 @@
 import type { RawCertificationProgram } from './content/schemas/certification-program'
 
+/**
+ * Program slugs are mainly used for stylistic tweaks. This type, and related
+ * components,  will need to expanded if additional programs are added.
+ */
+export type ProgramSlug =
```

Comment:
💭 follow up thought from my previous PR comment. If we want the slugs to be editable we may want to make this just a `string` to be more flexible. But not sure if we want to open up the slugs for editing.

Reply:
I see now why we want to type this since its used to generate css classes. So no need for action.